doc/releases.md

```diff
@@ -0,0 +1,51 @@
+
+This is the core, free, anonymous service available for anyone to use and which serves up more than 500 million badges every month!
+
+We do not have a fixed schedule for deploying updates to the Shields.io production environment, but we typically deploy the latest version at least once per week.
+
+More information about the production environment can be found [here][production hosting]
+
+## Shields server
+
+Some users may wish to host their own instance of Shields so we also make it possible for users to do so. This is particularly useful if you want to serve badges for resources that require authentication or are not exposed to the internet (e.g: inside a corporate network). A variety of options are available either by installing from source or using a docker image.
+
+### Important information
+
+This Shields server is the exact same codebase that powers the main Shields.io service; we do not have a separate self-hosted version of Shields. This means that the server codebase is geared towards the development, maintenance, and operation of the Shields.io service so there are a few things to note:
+
+- We often have to reject or de-prioritize feature requests that are only applicable for self-hosting and/or which may be problematic for the core Shields.io offering (e.g. requiring auth on the badge server)
+- We do not accept new badges that cannot be utilized with Shields.io (e.g. those that would always require user-specific authorization)
+- We do not do any additional testing nor validation of self-hosted instances (e.g. we test Shields.io with Jira Cloud badges, but we don't test a self-hosted Shields server against a self-hosted Jira server instance with auth)
+- We're happy to try to offer some tips and guidance to self-hosters when and where we can, but we don't have the ability to troubleshoot/debug/support/etc. self-hosted Shields servers
+  - Note that we may offer some sort of paid support arrangement in the future for those that would be interested
+
+### Server Releases
+
+We try to make it as easy as possible for users to self-host a Shields server so we publish a few releases of the server. Please be sure to refer to the [self hosting guide][self hosting] for a detailed walk through on how to spin up a server.
+
+- The server uses [Calendar Versioning](https://calver.org/). Tags of the form `server-YYYY-MM-DD` are server releases (these are the tags that are relevant to self-hosting users, e.g. [server-2021-02-01](https://github.com/badges/shields/releases/tag/server-2021-02-01)).
+- As well as [tags on GitHub](https://github.com/badges/shields/tags), server releases are also pushed to [DockerHub](https://registry.hub.docker.com/r/shieldsio/shields/tags). See the section on [Docker](#Docker) for more details.
```

Comment:
This is a dead link - there's no #Docker heading in this doc

Reply:
Ah, my lazy copy pasting w/o reading strikes again. Good catch